src/Microsoft.ApiDesignGuidelines.Analyzers/Core/UriParametersShouldNotBeStrings.cs

```diff
@@ -24,15 +26,103 @@ public abstract class UriParametersShouldNotBeStringsAnalyzer : DiagnosticAnalyz
                                                                              s_localizableMessage,
                                                                              DiagnosticCategory.Design,
                                                                              DiagnosticSeverity.Warning,
-                                                                             isEnabledByDefault: false,
+                                                                             isEnabledByDefault: true,
                                                                              description: s_localizableDescription,
-                                                                             helpLinkUri: null,     // TODO: add MSDN url
+                                                                             helpLinkUri: "https://msdn.microsoft.com/en-us/library/ms182174.aspx",
                                                                              customTags: WellKnownDiagnosticTags.Telemetry);
 
         public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => ImmutableArray.Create(Rule);
 
         public override void Initialize(AnalysisContext analysisContext)
         {
+            // this is stateless analyzer, can run concurrently
+            analysisContext.EnableConcurrentExecution();
```

Comment:
@srivatsn @jmarolf We should probably change the template analyzer project to now emit commented out code with a TODO for both these invocations. Something along the below lines:

```C#

// TODO: Configure concurrent execution callbacks for your analyzer.
//analysisContext.EnableConcurrentExecution(); 

// TODO: Configure generated code analysis for your analyzer.
//analysisContext.ConfigureGeneratedCodeAnalysis(GeneratedCodeAnalysisFlags.None);

```